roles/nova-data/tasks/main.yml

```diff
@@ -27,6 +27,17 @@
 
 - modprobe: name=nbd state=present
 
+- name: Turning off SMT mode for ppc64le
+  command: "ppc64_cpu --smt=off"
+  when: ansible_architecture == "ppc64le"
+
+- name: Adding SMT mode off in rc.local
+  lineinfile: 
+    dest: /etc/rc.local 
+    line: "ppc64_cpu --smt=off"
```

Comment:
Plan was to set ```smt=off``` at boot time(since it resets to a default value each time), hence have put it in ```rc.local```.
How about following approach.
1. writing an init script, ```/etc/init.d/ppc_smtoff.sh``` and
2. ```update-rc.d ppc_smtoff.sh defaults 99```
Another approach could be updating ```grub.conf```.
Which is the preferable file location ?